### ISSUE TITLE
Export data

### ISSUE THREAD

**Issue by joaquinsfy** [matt-FFFFFF/hassio-addon-teslamate#112]
it's possible  to export data in postgress to import in another instance?
Thanks

**Comment by matt-FFFFFF** [comment]
Yes. You have to expose the Postgres port in the Postgres addon and then you can connect to it using tools, e.g. https://www.pgadmin.org

**Comment by couzensmark** [comment]
Hi Matt,

Could you explain a little more on this please? I have cannot see how to export the database.

**Comment by matt-FFFFFF** [comment]
Have you seen #82? the process is the same
